test/parallel_exec.jl

```diff
@@ -11,6 +13,71 @@ elseif Base.JLOptions().code_coverage == 2
 end
 addprocs(3; exeflags=`$cov_flag $inline_flag --check-bounds=yes --depwarn=error`)
 
+# Test idleworker()
+let
+
+    @test Base.idleworker() == 2
+
+    r = nothing
+    function remote_sleep(t)
+        @async r = remotecall_fetch(sleep, Base.idleworker(), t)
+        yield()
+    end
+
+    w = workers()
+    @test length(w) == 3
+                                        t1 = now()
```

Comment:
Hi @yuyichao, I apologise if this looks weird to you.
I found that putting the temporal comments and assertions into a seperate column made it easier to understand.